web/models/terms.ex

```diff
@@ -70,4 +70,19 @@ defmodule CoursePlanner.Terms do
   defp non_deleted_query do
     from t in Term, where: is_nil(t.deleted_at)
   end
+
+  def notify_term_users(term, current_user, notification_type) do
+    term
+    |> get_enrolled_students()
```

Comment:
I think literally everyone except volunteers needs to be informed about the changes in the term